modules/ducktests/tests/ignitetest/tests/snapshot_test.py

```diff
@@ -0,0 +1,134 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Module contains discovery tests.
+"""
+from ducktape.mark.resource import cluster
+
+from ignitetest.services.ignite import IgniteService
+from ignitetest.services.ignite_app import IgniteApplicationService
+from ignitetest.services.utils.control_utility import ControlUtility
+from ignitetest.services.utils.ignite_configuration import IgniteConfiguration, DataStorageConfiguration
+from ignitetest.services.utils.ignite_configuration.data_storage import DataRegionConfiguration
+from ignitetest.services.utils.ignite_configuration.discovery import from_ignite_cluster
+from ignitetest.utils import ignite_versions
+from ignitetest.utils.ignite_test import IgniteTest
+from ignitetest.utils.version import DEV_BRANCH, IgniteVersion
+
+
+# pylint: disable=W0223
+class SnapshotTest(IgniteTest):
+    """
+    Test Snapshot.
+    """
+    NUM_NODES = 4
+
+    SNAPSHOT_NAME = "test_snap"
+
+    @cluster(num_nodes=NUM_NODES)
+    @ignite_versions(str(DEV_BRANCH))
+    def snapshot_test(self, ignite_version):
+        """
+        Test Snapshot.
+        1. Start of ignite cluster.
+        2. Activate cluster.
+        3. Load.
+        4. Idle verify dump_1.
+        5. Snapshot.
+        6. Load.
+        7. Idle verify dump_2.
+        8. Сhecking that dump_1 and dump_2 are different.
+        9. Stop ignite and replace db from snapshot.
+        10. Run ignite cluster.
+        11. Idle verify dump_3.
+        12. Checking the equality of dump_1 and dump_3.
+        """
+        data_storage = DataStorageConfiguration(default=DataRegionConfiguration(persistent=True))
+
+        ignite_config = IgniteConfiguration(
+            version=IgniteVersion(ignite_version),
+            data_storage=data_storage
+        )
+
+        service = IgniteService(self.test_context, ignite_config, num_nodes=self.NUM_NODES - 1)
+        service.start()
+
+        control_utility = ControlUtility(service, self.test_context)
+        control_utility.activate()
+
+        client_config = IgniteConfiguration(
+            client_mode=True,
+            version=IgniteVersion(ignite_version),
+            discovery_spi=from_ignite_cluster(service),
+        )
+
+        streamer = IgniteApplicationService(
+            self.test_context,
+            client_config,
+            java_class_name="org.apache.ignite.internal.ducktest.tests.UuidDataStreamerApplication",
+            params={
+                "cacheName": "test-cache",
+                "iterSize": 512 * 1024
+            }
+        )
+
+        load(streamer)
+
+        node = service.nodes[0]
+
+        control_utility.validate_indexes(check_assert=True)
+        control_utility.idle_verify(check_assert=True)
+        dump_1 = control_utility.idle_verify_dump(node, return_path=True)
+
+        self.logger.info(f'Path to dump_1 on {node.account.externally_routable_ip}={dump_1}')
```

Comment:
any reason to log this? 
is this a "phase logging"?

Reply:
fixed